R/pivot_wider_transition_risk_profile.R

```diff
@@ -0,0 +1,212 @@
+#' Calculate the indicator "transition risk profile"
+#'
+#' @param pivot_wider Logical. Pivot the output at company level to a wide
+#'   format?
+#'
+#' @return A data frame with the column `companies_id`, and the nested
+#'   columns`product` and `company` holding the outputs at product and company
+#'   level.
+#' @export
+#' @keywords internal
+#'
+#' @examples
+#' \dontrun{
+#' library(readr, warn.conflicts = FALSE)
+#' library(dplyr, warn.conflicts = FALSE)
+#' library(tiltToyData, warn.conflicts = FALSE)
+#' library(tiltTransitionRisk, warn.conflicts = FALSE)
+#'
+#' set.seed(123)
```

Comment:
Why is it important in this example to "freeze" random values? It seems the point of this function is to pivot columns, so the exact values of the columns doesn't seem important here.

Reply:
It's not important. Thanks! I removed it.